### ISSUE TITLE
Thank you and question...

### ISSUE THREAD

**Issue by BuddhaDiedLaughing** [jcbastosportela/logioptspp#14]
Thank you for this work.  Another obnoxious issue, for me, is the size of the package Logitech wants us to install just to get smooth scrolling. 2 or 300 MB is ridiculous.  And they still couldn't manage to get it to function reliably.  "SmoothScrolling" does it in a sub-2 MB package but it costs 20 bucks a year which is insane to me.

ANYWAY, I have a question:

I'm assuming I'll need Logitech's bloatware for this to work.  Is that correct?  Without it I tried running the exe but it didn't present me with anything just flashed the screen briefly.

Thanks again

**Comment by jcbastosportela** [comment]
Yes, indeed. This project simply intercepts calls that Logitech's software makes to Windows libs and lie about which app is in foreground.
